examples/with-segment-analytics/pages/_app.js

```diff
@@ -1,8 +1,29 @@
 import Page from '../components/Page'
+import Script from 'next/script'
+import * as snippet from '@segment/snippet'
+
+function renderSnippet() {
+  const opts = {
+    apiKey:
+      process.env.NEXT_PUBLIC_ANALYTICS_WRITE_KEY ||
+      'NPsk1GimHq09s7egCUlv7D0tqtUAU5wa',
+    // note: the page option only covers SSR tracking.
+    // Page.js is used to track other events using `window.analytics.page()`
+    page: true,
+  }
+
+  if (process.env.NEXT_PUBLIC_NODE_ENV === 'development') {
```

Comment:
```suggestion
  if (process.env.NODE_ENV === 'development') {
```
This should just be `NODE_ENV` for dead-code elimination to work correctly